js/index.js

```diff
@@ -1,6 +1,66 @@
 // Iteration 1: Names and Input
+//1.1
+const hacker1 = "Alexis";
+//1.2
+console.log(`The driver's name is ${hacker1}`);
+//1.3
+const hacker2 = "Alan";
+//1.4
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
-
+const longitud1 = hacker1.length;
+const longitud2 = hacker2.length;
+console.log (longitud1,longitud2)
+if (longitud1>longitud2) {
+  console.log (`The driver has the longest name, it has ${longitud1} characters.`)
+} else if (longitud1<longitud2){
+  console.log (`It seems that the navigator has the longest name, it has ${longitud2} characters.`)
+} else if (longitud1===longitud2){
+  console.log (`Wow, you both have equally long names, ${longitud1} characters!`)
+}
 
 // Iteration 3: Loops
+//3.1
+
+const driver = hacker1.toUpperCase().split("").join(" ").replace(/\s+/g, " ");
+console.log(driver);
+
+//3.2
+
+function reverseString(s){
+    return s.split("").reverse().join("");
+}
+reverseString(hacker2);
+console.log(reverseString(hacker2));
+
+//3.3
+
+const comparativaHackers = hacker1.localeCompare(hacker2)
+
+if (comparativaHackers === 1) {
```

Comment:
Aca podemos obviar la ultima 👀 
```suggestion
if (comparativaHackers === 1) {
if (hacker1.localeCompare(hacker2) === -1) {
  console.log("The driver's name goes first.")
}
else if (hacker1.localeCompare(hacker2) === 1) {
  console.log("Yo, the navigator goes first definitely.")
}
else {
  console.log("What?! You both have the same name?")
} 
```